src/components/asset-list/RecyclerAssetList2/core/RawRecyclerList.tsx

```diff
@@ -182,6 +172,15 @@ const RawMemoRecyclerAssetList = React.memo(function RawRecyclerAssetList({
       refreshControl={disablePullDownToRefresh ? undefined : <RefreshControl />}
       renderAheadOffset={1000}
       rowRenderer={rowRenderer}
+      canChangeSize={type === 'wallet'}
+      layoutSize={
```

Comment:
Likely worth defining the height/width object outside of the component so it doesn't trigger updates.

Reply:
Referring to this:

```ts
type === 'wallet'
  ? {
    height: deviceUtils.dimensions.height,
    width: deviceUtils.dimensions.width,
  }
  : undefined
```